Webex Connect Flows/v3.0/Template/Event Handling Workflows/README.md

```diff
@@ -0,0 +1,155 @@
+1. If you want to run a flow to run only for a particular channel let's say LiveChat then you can specify condition like webex.mediaChannel equals web.
+
+2. If you want to run a flow only for some specific asset then we can use  the "webex.destination" field which will help us to determine the asset.
+
+Values of "webex.destination" for different channels: -
+* LiveChat :- App ID
+* Whatsapp :- App ID
+* Facebook :- App ID
+* Email :- App ID
+* SMS :- Service Number
+
+The App ID  of a particular asset can be obtained from the assets→apps page in Webex Connect portal.
+
+<img width="800" alt="App" src="../../images/App.png">
+
+The Service Number of a particular sms asset can be obtained from the assets→Numbers page in Webex Connect portal.
+
+<img width="800" alt="App" src="../../images/Numbers.png">
+
+If we want to run the flow for multiple assets then additional new "equals" condition must be added with the OR operator.
+
+<img width="800" alt="OrCondition" src="../../images/ORCondition.png">
+
+If we don't want to run the flow for a specific asset then we can use "notequals" condition.
+
+<img width="800" alt="notequalssinglecondition" src="../../images/notequalssinglecondition.png">
+
+If we don't want to the run the flow for multiple assets then additional "notequals" condition must be added with the AND operator.
+
+<img width="800" alt="NotEquals" src="../../images/NotEquals.png">
+
+For more details on adding conditions please refer https://help.imiconnect.io/docs/start-node#trigger-conditions-in-start-node
+
+### Using variables or details in event flows
+
+Evaluate node is extracting variables from json. 
+In evaluate node we are defining and assigning all the predefined system variables.
+
+<img width="800" alt="Evaluate" src="../../images/Evaluate.png">
+
+We also have helper method to extract flow and global variables which are basically set in the main flows via set variable node.
+
+<img width="800" alt="Helper method" src="../../images/Helper method.png">
+
+Note : Predefined System Variables and Extracted variables can be used to do any custom logic like screen pop or http calls.
+Refer to [Screen Pop Readme](../../Sample/Usage of Screen Pop in Flows/README.md)
+All the sample event flows demonstrate screen pop usage though they can be used for building other custom logic also.
+
+
+## Migration Strategy
+* Flow admins can manually migrate the existing main flows for each asset to V3.0 at their pace.
+* The shared flows can be manually migrated to the V3.0 versions only when all the main flows for each asset are migrated already.
+* If the org has migrated only a few of their main flows to V3.0, shared V2.x flows should have the condition to prevent running for contacts created via V3.0 flows. Similarly, V3.0 shared flows should have the condition to run only for the assets migrated to V3.0.
+* For migration, refer to the steps mentioned in 
+  [Migration Strategy](Migration Strategy.pdf)
```

Comment:
The link for Migration Strategy.pdf is not rendered correctly. Shows in plaintext.
<img width="1069" alt="Screenshot 2023-05-09 at 12 54 25 PM" src="https://user-images.githubusercontent.com/40895295/237024436-d049624f-acd5-4698-a889-ae8e4a72f7d6.png">

Reply:
Done